Pipfile

```diff
@@ -23,6 +23,7 @@ aiocontextvars = "*"
 sentry-sdk = {extras = ["flask"],version = "*"}
 confluent-kafka = "*"
 thoth-messaging = "*"
+openshift = "==0.11.0"
```

Comment:
we should pin this in common https://github.com/thoth-station/common/blob/b93627edfb51e1816995c24e1528366c678e04d8/Pipfile#L13
as we have kubernetes pinned down there as well .
this would make it easier for other apps as well.

Reply:
Fair point :+1: opened https://github.com/thoth-station/common/pull/1126